Add check for ReplicationTable online status in Monitor

tchaie opened the pull request:
Closes #2653

ctubbsii commented on server/monitor/src/main/java/org/apache/accumulo/monitor/rest/replication/ReplicationResource.java:
```diff
@@ -149,6 +149,12 @@ public List<ReplicationInformation> getReplicationInformation()
       });
     }
 
+    //Ensure replication table is online before attempting to create BatchScanner
+    if (!ReplicationTable.isOnline(client)) {
+        log.error("Replication table is offline");
```
I suspect the formatter will add a space when it builds. I enabled a build on this PR, so we'll find out.

The message can also be a debug instead of an error, since this is probably a normal situation (it's not a problem if the user clicks the page).

```suggestion
    // Ensure replication table is online before attempting to create BatchScanner
    if (!ReplicationTable.isOnline(client)) {
        log.debug("Replication page requested, but replication table is offline");
```

FerFlovi replied:
```suggestion
    //Ensure replication table is online before attempting to create BatchScanner
    if (!ReplicationTable.isOnline(client)) {
        log.error("Replication table is offline");
```

tchaie replied:
@ctubbsii In the Monitor, the Replication page already indicates "Replication table is offline". Is this sufficient?

ctubbsii replied:
If that's already the case, then nothing needs to be done in that regard.

EdColeman replied:
Maybe better as a separate issue, but is that how the message reads? It may be more informative if the message indicated something along the lines of:
 
```Replication is disabled - the replication table is offline```

My thinking is that not using replication is more common. And having the replication off-line is the expected (desired) state.  Clarifying that may help if someone somehow interrupted off-line as bad.  Only people using replication should care.

ctubbsii reviewed (changes requested):
Thanks for the PR @tchaie . I made a few comments on the change.

I haven't run this, but it looks like it should work. I do wonder what the user experience is, though... what will the user see on the page? This PR looks like it will get rid of the stack trace/error, but it might be nice to make sure the page shows "Replication is currently disabled" or a similar message when the user does navigate to the page.

FerFlovi reviewed (commented):
Duplicado de #

ctubbsii commented:
@tchaie I think needs to be rebased on main, and updated to exclude some of the unrelated console log changes. I think those are related to your other PR, and don't need to be included here.

tchaie commented:
> @tchaie I think needs to be rebased on main, and updated to exclude some of the unrelated console log changes. I think those are related to your other PR, and don't need to be included here.

Rebased and reverted commit containing console log updates.
